ddtrace/tracer/span.go

```diff
@@ -81,7 +81,7 @@ func (s *span) SetTag(key string, value interface{}) {
 		return
 	}
 	if key == ext.Error {
-		s.setTagError(value)
+		s.setTagError(value, true)
```

Comment:
Would you mind making this change here?
```go
withDebugStack := true
s.setTagError(value, withDebugStack)
```
I know it's redundant but this way people don't have to go to the method to see what `true` means. I saw this in another project and liked it.

Reply:
Would it look better if I change the other way?
```
type tagErrorArgs struct {
    value interface{} 
    debugStack bool
}
s.setTagError(tagErrorArgs{
  value: value, debugStack: true
})
```

Reply:
I prefer what we have now over this suggestion. You can chose between what we have now and my suggestion. I am fine with either. This latest is a bit overkill at the moment imho.

Reply:
Let's leave it as it is now.